python/ray/autoscaler/README.rst

```diff
@@ -0,0 +1,24 @@
+Automated cluster setup and auto-scaling support (Experimental)
```

Comment:
How about moving this to `doc/source/` so we can link to it from readthedocs?

Reply:
Done